src/annotator/components/ContentInfoBanner.js

```diff
@@ -97,9 +103,14 @@ export default function ContentInfoBanner({ info }) {
             'min-w-0 whitespace-nowrap overflow-hidden text-ellipsis shrink font-medium'
           )}
         >
-          <span title={info.item.title} data-testid="content-item-title">
-            {info.item.title}
-          </span>
+          <LinkUnstyled
```

Comment:
`LinkUnstyled` is from the updated UI toolkit set and does what it sounds like: creates a link (`<a>`) but doesn't style it. This is per the intent of linking the item title but not making it "look" like a link.

Reply:
I was going to ask about this. This strikes me as strange but ok? Is there any other kind of affordance we could add to tip people off like 🔗 or something?